### DISCUSSION TITLE
Tom's Hardware posted an article, Microsoft to break third party apps like EP with 24H2

### DISCUSSION THREAD

**Discussion by jab8356** [valinet/ExplorerPatcher#3105] [General]
https://www.tomshardware.com/software/operating-systems/microsoft-stifles-third-party-interface-customization-apps-in-windows-11-version-24h2

**Comment by pyrates999** [comment]
Microsoft is attempting to block them because it is an unreleased version of windows and wants you to be testing it without EP or StartAllBack installed.

**Comment by Hexality** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/17398632/f280a169-a005-414e-85df-711c0f95d9b4)
> From [`@Endermanch`](https://x.com/endermanch) on [X](https://x.com/endermanch/status/1775639423691964520)

![image](https://github.com/valinet/ExplorerPatcher/assets/17398632/9c6fbcb2-29c2-4225-852f-307d28cba3ee)
> From `@quoyque` on `Unofficial Patch for Metro for Steam` discord server.

Bet Start11, UltraUXPatcher and SecureUXPatcher are the next ones to be blocked...

**Reply by pecanpie** [reply]
All you have to do is rename the setup file.

**Reply by Hexality** [reply]
that's just a shitty workaround, there are already rumors that microsoft might start to block certain certificates to block the apps again.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3085

please close this.

https://fxtwitter.com/StartIsBack/status/1775660727719928173

You're not supposed to use ExplorerPatcher on Dev/Canary at the moment.

**Reply by Hexality** [reply]
this is a discussion not a issue.
